src/lib/memory/blob_manager.cpp

```diff
@@ -2,7 +2,7 @@
 
 namespace homeobject {
 
-ShardIndex& MemoryHomeObject::_find_index(shard_id shard) {
+ShardIndex& MemoryHomeObject::_find_index(shard_id shard) const {
     auto index_it = _in_memory_index.find(shard);
     RELEASE_ASSERT(_in_memory_index.end() != index_it, "Missing BTree!!");
     return *index_it->second;
```

Comment:
NPE here if shard doesnt exists

Reply:
NPE?

Reply:
nevermind,  you catch that by RELEASE_ASSERT.